tests/integration/test_wazuh_db/test_wdb_backup_configs.py

```diff
@@ -0,0 +1,229 @@
+'''
+copyright: Copyright (C) 2015-2022, Wazuh Inc.
+           Created by Wazuh, Inc. <info@wazuh.com>.
+           This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+type: integration
+
+brief: Wazuh-db is the daemon in charge of the databases with all the Wazuh persistent information, exposing a socket
+       to receive requests and provide information. Wazuh-db has the capability to do automatic database backups, based
+       on the configuration parameters. This test, checks the proper working of the backup configuration and the
+       backup files are generated correctly.
+
+tier: 0
+
+modules:
+    - wazuh_db
+
+components:
+    - manager
+
+daemons:
+    - wazuh-db
+
+os_platform:
+    - linux
+
+os_version:
+    - Arch Linux
+    - Amazon Linux 2
+    - Amazon Linux 1
+    - CentOS 8
+    - CentOS 7
+    - CentOS 6
+    - Ubuntu Focal
+    - Ubuntu Bionic
+    - Ubuntu Xenial
+    - Ubuntu Trusty
+    - Debian Buster
+    - Debian Stretch
+    - Debian Jessie
+    - Debian Wheezy
+    - Red Hat 8
+    - Red Hat 7
+    - Red Hat 6
+
+references:
+    - https://documentation.wazuh.com/current/user-manual/reference/daemons/wazuh-db.html
+
+tags:
+    - wazuh_db
+'''
+
+import os
+import pytest
+import subprocess
+import time
+import numbers
+
+from wazuh_testing.tools.configuration import load_wazuh_configurations
+from wazuh_testing.tools.services import control_service
+from wazuh_testing.tools.file import recursive_directory_creation, remove_file, truncate_file
+from wazuh_testing.tools.monitoring import FileMonitor, generate_monitoring_callback
+from wazuh_testing.tools import LOG_FILE_PATH, WAZUH_PATH
+
+
+# Marks
+pytestmark = pytest.mark.tier(level=0)
```

Comment:
Could you use marks defined on deps\wazuh_testing\wazuh_testing\modules\__init__.py, please?